Jenkinsfile

```diff
@@ -1,4 +1,4 @@
-@Library('defra-library@3.0.1')
+@Library('defra-library@4.4.1')
```

Comment:
Do you definitely need this exact version? If you're happy for the latest version (without changing major version), you could do `'defra-library@4`. You can tie to a minor version by doing `'defra-library@4.4'`

Reply:
Excellent idea, changed it to `4`